includes/class-llms-blocks.php

```diff
@@ -169,7 +169,7 @@ public function init() {
 		add_action( 'add_meta_boxes', array( $this, 'remove_metaboxes' ), 999, 2 );
 
 		global $wp_version;
-		$filter = version_compare( $wp_version, '5.8-alpha.1', '>=' ) ? 'block_categories_all' : 'block_categories';
+		$filter = version_compare( $wp_version, '5.8-src', '>=' ) ? 'block_categories_all' : 'block_categories';
```

Comment:
version / changelogs

Reply:
I can't believe I missed that. Thank you.